src/app/interfaces/interfaces.ts

```diff
@@ -0,0 +1,17 @@
+export interface Product {
+    price: number
+}
+
+export interface Order {
+    customerId: number,
+    quantity: number,
+    status: OrderStatus,
+    totalCost: number
+}
+
+type OrderStatus = 'places' | 'pending' | 'approved' | 'complete' | 'denied' | 'cancelled';
+
+export interface Customer {
+    name: string,
+    banking_details: string
+}
```

Comment:
Please can you move these into there own files
also for format sake,
order.d.ts

And types should live in there own folder

Reply:
done